spec/unit/provider/user/solaris_spec.rb

```diff
@@ -31,15 +36,6 @@
     p
   end
 
-  supported_useradd_options = {
-    'comment' => "-c",
-    'gid' => "-g",
-    'uid' => "-u",
-    'shell' => "-s"
-  }
-
-  include_examples "a useradd-based user provider", supported_useradd_options
```

Comment:
In order to get the tests to pass I had to remove this.   the base_useradd tests were running, which were asserting that the commands ran to lock and unlock users were variations of the `usermod` command, not the `passwd` commands I have added in this change.